external/emulation/emulation/hooks/config.py

```diff
@@ -0,0 +1,61 @@
+import dataclasses
+import logging
+from typing import Optional
+
+import dacite
+import yaml
+from emulation.hooks import microphysics, monitor
+from emulation.hooks.microphysics import _load_tf_model
+
+logger = logging.getLogger("emulation")
+
+
+def error_on_call(e):
+    def func(state):
+        raise e
+
+    return func
+
+
+def do_nothing(state):
+    pass
+
+
+@dataclasses.dataclass
+class ModelConfig:
+    path: str
+
+
+@dataclasses.dataclass
+class EmulationConfig:
+    model: Optional[ModelConfig] = None
+    storage: Optional[monitor.StorageConfig] = None
+
+    def build_model_hook(self):
+        if self.model is None:
+            return error_on_call(
```

Comment:
This behavior is modified in the next commit. This breaks the `NoModel` behavior.